radiomics/base.py

```diff
@@ -63,13 +61,19 @@ def __init__(self, inputImage, inputMask, **kwargs):
       self.logger.warning('Missing input image or mask')
       return
 
+  def _initLesionWiseCalculation(self):
```

Comment:
Not immediately clear from the name of the function what is its intent. When I just look at the name, I think that it means it can take an image with multiple lesions, and calculate features for each one, but it does not look like that's what it is actually doing. May also make sense to add some documentation about the function, especially considering it is reimplemented in derived classes.